server/controllers/meeting.js

```diff
@@ -0,0 +1,45 @@
+const Meeting = require("../models/Meeting");
+const asyncHandler = require("express-async-handler");
+
+// @route POST /meeting
+// @desc Create a meeting
+// @access Private
+exports.createMeeting = asyncHandler(async (req, res, next) => {
+  try {
+    const meeting = await Meeting.create(req.body);
```

Comment:
In the future, you should consider maybe doing input validation either on the schema or via middleware.